stdlib/public/SwiftShims/UnicodeShims.h

```diff
@@ -489,27 +488,6 @@ SWIFT_RUNTIME_STDLIB_API
 __swift_int32_t __swift_stdlib_ubrk_following(__swift_stdlib_UBreakIterator *bi,
                                               __swift_int32_t offset);
 
-SWIFT_RUNTIME_STDLIB_API
-__swift_stdlib_UBool
-__swift_stdlib_unorm2_hasBoundaryBefore(const __swift_stdlib_UNormalizer2 *,
-                                        __swift_stdlib_UChar32);
-
-SWIFT_RUNTIME_STDLIB_API
-const __swift_stdlib_UNormalizer2 *
-__swift_stdlib_unorm2_getNFCInstance(__swift_stdlib_UErrorCode *);
-
-SWIFT_RUNTIME_STDLIB_API
-__swift_int32_t
-__swift_stdlib_unorm2_normalize(const __swift_stdlib_UNormalizer2 *,
-                                const __swift_stdlib_UChar *, __swift_int32_t,
-                                __swift_stdlib_UChar *, __swift_int32_t,
-                                __swift_stdlib_UErrorCode *);
-
-SWIFT_RUNTIME_STDLIB_API
-__swift_int32_t __swift_stdlib_unorm2_spanQuickCheckYes(
-    const __swift_stdlib_UNormalizer2 *, const __swift_stdlib_UChar *,
-    __swift_int32_t, __swift_stdlib_UErrorCode *);
-
```

Comment:
I'm not sure what you mean by this, do these definitions have declarations? If so, yes they do, and git doesn't do a great job of showing that the current file where these are in are `UnicodeNormalization.cpp` but I wanted to use this name for the native stuff, so I moved the ICU declarations in `UnicodeShims.cpp` and removed these.

Reply:
I was referring to the contents of `stdlib/public/SwiftShims/UnicodeShims.h`. If we're removing stuff, we probably want to remove the `.h` file entry too so that this can be a compile-time error instead of a link-time error.